doc/man/julia.1

```diff
@@ -188,7 +191,7 @@ Set the optimization level (level 3 if `-O` is used without a level)
 Set a lower bound on the optimization level
 
 .TP
--g {0,1*,2}
+-g, --debug-info={0,1*,2}
```

Comment:
It'd be great to also standardize on `|`s to delineate options — especially since some flags values may contain `,`s!

```suggestion
-g, --debug-info={0|1*|2}
```